server/container/guice/cassandra-rabbitmq-guice/src/test/java/org/apache/james/modules/blobstore/BlobStoreConfigurationTest.java

```diff
@@ -223,4 +223,33 @@ void fromShouldReturnConfigurationWhenBlobStoreImplIsSupportedAndHasExtraSpaces(
                 .getName())
             .isEqualTo(CASSANDRA);
     }
+
+    @Test
+    void cacheEnabledShouldBeTrueWhenSpecified() {
```

Comment:
`cacheEnabled` or `cacheEnable`? as the conf parameter is `cache.enable` (applies for tests after too if agreed)

Reply:
But he getter is named `cacheEnabled` and this is the property tested in the left part of the assertion.